lib/blocktree/blocktree.go

```diff
@@ -23,17 +23,15 @@ type BlockTree struct {
 	root   *node
 	leaves *leafMap
 	sync.RWMutex
-	nodeCache map[Hash]*node
-	runtime   *sync.Map
+	runtime *sync.Map
 }
 
 // NewEmptyBlockTree creates a BlockTree with a nil head
 func NewEmptyBlockTree() *BlockTree {
 	return &BlockTree{
-		root:      nil,
-		leaves:    newEmptyLeafMap(),
-		nodeCache: make(map[Hash]*node),
-		runtime:   &sync.Map{}, // map[Hash]runtime.Instance
+		root:    nil,
+		leaves:  newEmptyLeafMap(),
+		runtime: &sync.Map{}, // map[Hash]runtime.Instance
```

Comment:
better to put this comment in definition of BlockTree. I am assuming that this map would always have `map[Hash]runtime.Instance` type